### ISSUE TITLE
module "nnlm-ko-dim128" should be compatible with TF2.0 

### ISSUE THREAD

**Issue by jason9693** [tensorflow/hub#306]
I tried to using [nnlm-ko-dim-128](https://tfhub.dev/google/nnlm-ko-dim128-with-normalization/1) as a keras layer.
Unfortunately, this module is compatible with  TF 1.x and also 1.x module format is different with 2.0 ( the result of 'saved_model.load()' is callable in 2.0 module;; )

So, can you
1. provide 2.0 module as soon as possible? We want this module in a short time.

OR,

2. explain how to convert the module to 2.0 format? I can't find any document for convert 1.x module to 2.0 module. 

OR,

3. explain how to create a module in 2.0 format? [Creating module document](https://www.tensorflow.org/hub/creating) is explained only 1.x version, So we can't know and export to 2.0 format. I'm willing to contribute code, binary file or any formats.

Thank you.

**Comment by arnoegw** [comment]
Hi Jason, both TF2 and its support in TF-Hub are still being worked on. As part of that, we are working both on publishing more modules in TF2 format as well as making modules in the Hub/TF1 format usable in TF2, with hub.KerasLayer.

If you are in a rush to get something off the ground with TF2, you could try the hacky approach from issue https://github.com/tensorflow/hub/issues/281
